app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -816,6 +817,26 @@ class HomeFragment : Fragment() {
         // triggered to cause an automatic update on warm start (no tab selection occurs). So we
         // update it manually here.
         requireComponents.useCases.sessionUseCases.updateLastAccess()
+
+        if (shouldEnableWallpaper()) {
+            _binding?.sessionControlRecyclerView?.viewTreeObserver?.addOnGlobalLayoutListener(
```

Comment:
I think we should still use `binding`, otherwise, we might miss some regressions if this ends up being called at the wrong time in the lifecycle.

Reply:
`binding` using `!!` that causes null pointer exceptions in some cases, using  `_binding` we have control over when the `binding` is not available.

https://github.com/mozilla-mobile/fenix/blob/76fb147ed87c32f37b6b92db1a0d0b3541308d86/app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt#L139

Reply:
Are there specific cases you hit NPEs with here? Otherwise, I think we can just use `binding`, as that's what's used throughout the app. It should be safe between `onCreateView` and `onDestroyView` according to the [docs](https://developer.android.com/topic/libraries/view-binding#fragments)